src/montagu_api.py

```diff
@@ -0,0 +1,31 @@
+import requests
+
+
+class MontaguAPI:
+    def __init__(self, base_url, username, password):
+        self.base_url = base_url
+        auth_url = self.build_url('authenticate')
+        data = 'grant_type=client_credentials'
+        auth = (username, password)
+        headers = {'Content-Type': 'application/x-www-form-urlencoded'}
+        response = requests.post(auth_url, data=data, auth=auth,
+                                 headers=headers)
+        if response.status_code != 200:
+            msg = 'Unexpected status code: {}. Unable to authenticate.'
```

Comment:
Do you want to start on the good foot and test these two lines?

Reply:
It was actually a slightly different 2 lines, but point taken!

Reply:
oops sorry